### ISSUE TITLE
AIP 143 time-field-type incorrectly flags valid Date usages

### ISSUE THREAD

**Issue by amosyuen** [googleapis/api-linter#463]
https://linter.aip.dev/142/time-field-type incorrectly flags valid date usages.

Examples:

```
google.type.Date start_date = 1;
google.type.Date end_date = 2;
```

**Comment by odsod** [comment]
Submitted a PR proposing a fix for this. Cheers

**Comment by lukesneeringer** [comment]
Fixed by #470.
